dev/llc.sh

```diff
@@ -0,0 +1,332 @@
+    -e KAFKA_CONTROLLER_LISTENER_NAMES=CONTROLLER \
+    -e KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,BROKER:PLAINTEXT,EXTERNAL:PLAINTEXT \
+    -e KAFKA_PROCESS_ROLES='controller,broker' \
+    -e KAFKA_NODE_ID=1 \
+    -e KAFKA_CONTROLLER_QUORUM_VOTERS='1@0.0.0.0:9093' \
+    -e KAFKA_LOG_DIRS='/tmp/kraft-combined-logs' \
+    -e CLUSTER_ID=ciWo7IWazngRchmPES6q5A== \
+    -v $llr_dir/kafka:/var/lib/kafka/data \
+    confluentinc/cp-kafka:7.5.0
+    echo "Kafka: ✅"
+}
+
+minio_hostname=langstream-minio
+
+install_minio() {
+
+  if [ "$(docker ps -q -f name=$minio_hostname)" ]; then
+    echo "Minio: ✅"
+    return
+  fi
+
+  if [ "$(docker ps -a -q -f name=$minio_hostname)" ]; then
+    docker rm -f $minio_hostname
+  fi
+  docker run \
+   -d \
+   --name $minio_hostname \
+   -p 9090:9090 \
+   -v $llr_dir/minio:/data \
+   quay.io/minio/minio:latest \
+   server /data --console-address :9090
+   echo "Minio: ✅"
+}
+configure_cli() {
+  control_plane_url=$(start_port_forward langstream-control-plane)
+  api_gateway_url=$(start_port_forward langstream-api-gateway)
+  run_cli configure webServiceUrl $control_plane_url
+  run_cli configure apiGatewayUrl $api_gateway_url
+  run_cli configure tenant default
+  echo "CLI: ✅"
+}
+
+run_cli() {
+  LANGSTREAM_CLI_CONFIG=$langstream_cli_config langstream "$@"
+}
+start() {
+  local dev="false"
+  local build="false"
+  while [[ "$#" -gt 0 ]]; do
+      case $1 in
+          --dev) dev="true"; shift ;;
+          --build) build="true"; shift ;;
+          *) echo "Unknown parameter passed: $1"; exit 1 ;;
+      esac
+  done
+  start_minikube
+  install_langstream "$dev" "$build"
+  install_kafka
+  install_minio
+  configure_cli
+}
+delete() {
+  docker rm -f $kafka_hostname
+  docker rm -f $minio_hostname
+  delete_minikube
+  configure_cli
+}
+
+handle_load() {
+  local image=$1
+  if [[ $image == "control-plane" ]]; then
+    image=langstream/langstream-control-plane:latest-dev
+  elif [[ $image == "deployer" ]]; then
+    image=langstream/langstream-deployer:latest-dev
+  elif [[ $image == "api-gateway" ]]; then
+    image=langstream/langstream-api-gateway:latest-dev
+  elif [[ $image == "cli" ]]; then
+    image=langstream/langstream-cli:latest-dev
+  elif [[ $image == "runtime" ]]; then
+    image=langstream/langstream-runtime:latest-dev
+  fi
+  load_image $image
+  label=""
+  if [[ $image == *"langstream-control-plane"* ]]; then
+    label=langstream-control-plane
+  elif [[ $image == *"langstream-deployer"* ]]; then
+    label=langstream-deployer
+  elif [[ $image == *"langstream-api-gateway"* ]]; then
+    label=langstream-api-gateway
+  elif [[ $image == *"langstream-cli"* ]]; then
+    label=langstream-client
+  fi
+  if [ ! -z "$label" ]; then
+    kubectl_cmd -n $k8s_namespace delete pod -l app.kubernetes.io/name=$label
+    echo "Pod $label: ✅"
+  fi
+}
+build_no_restart() {
+  eval $(minikube_cmd docker-env)
+  ./docker/build.sh "$@" &> /dev/null
```

Comment:
please note that now docker/build.sh runtime is broken.
because now the runtime-impl module doen't depend anymore on the "agents", it is only copying the .nar files, so Maven with "-am" doesn't rebuild them.

in case of "runtime" we have to rebuild manually more stuff with "mvn clean install -DskipTests"

Reply:
no need to fix it in this patch, it is unrelated